src/Network/HTTP/Media/MediaType/Arbitrary.hs

```diff
@@ -12,11 +12,11 @@ module Network.HTTP.Media.MediaType.Arbitrary () where
 
 import Prelude hiding (concat)
 
-import Control.Applicative ((<*>))
+import Control.Applicative ()
```

Comment:
We don't want spurious classes from an otherwise unused module.  I wonder why this is no longer necessary.

Reply:
reverted. Maybe we can have chat about this since the compiler warnings are not fun to have.